odl/tomo/operators/xray_trafo.py

```diff
@@ -39,19 +41,22 @@
 
 _SUPPORTED_BACKENDS = ('astra', 'astra_cpu', 'astra_cuda')
 
-__all__ = ('DiscreteXrayTransform', 'DiscreteXrayTransformAdjoint')
+__all__ = ('XrayTransform', 'XrayTransformAdjoint',)
```

Comment:
Same question as several times before. Is the adjoint an operator in its own right? I think so. We should call it `XrayBackProjection` or something similar.